packages/python-filelock/python-filelock.spec

```diff
@@ -0,0 +1,47 @@
+# Created by pyp2rpm-3.3.3
+%global pypi_name filelock
+
+Name:           python-%{pypi_name}
+Version:        3.0.12
+Release:        1%{?dist}
+Summary:        A platform independent file lock
+
+License:        Public Domain <http://unlicense.org>
```

Comment:
I think the correct SPDX identifier is `Unlicense`: https://spdx.org/licenses/Unlicense.html